formfiller/src/main/java/com/diyasylum/formfiller/application/models/AddressWithDates.java

```diff
@@ -0,0 +1,108 @@
+package com.diyasylum.formfiller.application.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+
+public class AddressWithDates {
+  private final String streetName;
+  private final String streetNumber;
+  private final String cityOrTown;
+  private final String departmentProvinceOrState;
+  private final String country;
```

Comment:
Do we want to make this an enum or anything? Or do validation in some other way? Or just not care?

Reply:
In the FE we'd probably want to have an autofill or dropdown with all the countries. I'm not compelled to do further validation on the BE assuming it's implemented that way on FE. It does look like there are some packages with a country enum we could use if you prefer.

Reply:
This is fine as is